public/manifest.json

```diff
@@ -1,63 +1,29 @@
 {
-  "name": "You.com | Search, chat, and create with AI",
+  "name": "You.com: Default AI Search & Chat",
   "manifest_version": 3,
-  "version": "0.0.0.6",
-  "description" : "Experience the future of search with generative AI",
-  "action": {
-    "default_popup": "index.html",
-    "default_title": "Open the popup"
-  },
+  "version": "2.0.0",
+  "description": "Experience the future of search with generative AI",
   "content_scripts": [
     {
       "matches": ["<all_urls>"],
       "js": ["./static/js/content.js"]
     }
-  ], 
-  "declarative_net_request" : {
-    "rule_resources" : [
-        {
-        "id": "ruleset_1",
-        "enabled": false,
-        "path": "rules_1.json"
-        },
-        {
-          "id": "ruleset_2",
-          "enabled": true,
-          "path": "rules_2.json"
-        },
-        {
-          "id": "ruleset_3",
-          "enabled": false,
-          "path": "rules_3.json"
-        },
-        {
-          "id": "ruleset_4",
-          "enabled": false,
-          "path": "rules_4.json"
-        }
-      ]
-    },
-  "permissions": [
-    "declarativeNetRequestWithHostAccess", "storage"
-  ], 
-  "host_permissions": [
-  "*://*/*"
   ],
+  "permissions": ["declarativeNetRequestWithHostAccess", "storage"],
```

Comment:
i don't think we need declarativeNetRequest as a perm anymore since we removed those rules